docs/api-reference/benchmarks.mdx

```diff
@@ -0,0 +1,110 @@
+---
```

Comment:
This should not be part of our API docs.

Reply:
Right. Which folder do you think it would belong in most? I have had many users tell us our benchmarks are hard to find. Perhaps `welcome`/?